packages/aws-cdk/lib/init-templates/app/csharp/src/%name.PascalCased%/%name.PascalCased%.template.csproj

```diff
@@ -8,10 +8,9 @@
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
     <PackageReference Include="Amazon.CDK" Version="%cdk-version%-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.IAM" Version="%cdk-version%-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.SNS" Version="%cdk-version%-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.SNS.Subscriptions" Version="%cdk-version%-devpreview" />
-    <PackageReference Include="Amazon.CDK.AWS.SQS" Version="%cdk-version%-devpreview" />
+
+    <!-- jsii Roslyn analyzers -->
+    <PackageReference Include="Amazon.Jsii.Analyzers" Version="*" PrivateAssets="all" />
```

Comment:
Are we again trying this out? I would vote against introducing this now.

Reply:
Making them into a comment instead.